code/modules/events/viral_outbreak.dm

```diff
@@ -8,7 +8,7 @@ datum/event/viral_outbreak/setup()
 	severity = rand(2, 4)
 
 datum/event/viral_outbreak/announce()
-	command_announcement.Announce("Confirmed outbreak of level 7 biohazard aboard the [location_name()]. All personnel must contain the outbreak.", "Biohazard Alert", new_sound = GLOB.using_map.level_x_biohazard_sound(7))
+	command_announcement.Announce("Confirmed containment breach of KETER class object. All security personnel must recontain the object.", "Biohazard Alert", new_sound = GLOB.using_map.level_x_biohazard_sound(7))
```

Comment:
this doesn't make sense, why is a virus KETER

Reply:
its the blob you dip

Reply:
o.